modules/social_features/social_profile/modules/social_profile_fields/social_profile_fields.install

```diff
@@ -20,6 +20,15 @@ function social_profile_fields_install() {
   _social_profile_fields_update_search_index();
   _social_profile_fields_update_search_index('social_all');
   _social_profile_fields_nationalities();
+
+  $permissions = [
```

Comment:
@Macsym2018 And what about create/edit/delete terms permissions?
We actually need to make sure they are set for SM.

Reply:
@ribel Added